integration-tests/test_webhdfs.py

```diff
@@ -7,15 +7,58 @@
 #
 """
 Sample code for WebHDFS integration tests.
-Requires hadoop to be running on localhost, at the moment.
 """
+import json
+import os
 import smart_open
+from smart_open.webhdfs import WebHdfsException
+import pytest
 
-with smart_open.smart_open("webhdfs://localhost:50070/user/root/input/core-site.xml") as fin:
-    print(fin.read())
+_SO_WEBHDFS_BASE_URL = os.environ.get("SO_WEBHDFS_BASE_URL")
```

Comment:
Please explain the significance of this environment variable in the docstring for this script.

Also, include instructions on how to run the test. Is there anything else the user needs to set up, like other environment variables or software?

Reply:
done